HomeTask3.playground/Contents.swift

```diff
@@ -0,0 +1,67 @@
+import Darwin
+//Создать массив из кортежей (Int, String).
+
+let tuplesArray = [(1, "My name ist Valdemar"), (3, "Three"), (12, "Elizabeth"), (88, "Hello")]
+var tuplesArrayWithSquaredNumbers : [(Int, String)] = []
+
+//1. Пройти по элементам с помощью цикла for-in, создав Range от 0 до количества элементов массива, и возвести все Int в квадрат. (Лучше создать новый массив и результаты записать туда).
+
+for range in 0..<tuplesArray.count {
+    let squaring = powNumber(tuplesArray[range].0)
+    
+    tuplesArrayWithSquaredNumbers.insert((squaring,tuplesArray[range].1), at: tuplesArrayWithSquaredNumbers.endIndex)
+}
+
+func powNumber(_ number: Int, factor: Int = 2) -> Int {
+    Int(pow(Double(number), Double(factor)))
+}
+
+//2.Пройти по элементам изначального массива с помощью цикла for-in, перебирая элементы, и записать в новый массив только четные элементы (не по порядку четные, а с четными Int’ами). (* Написать альтернативный вариант с функцией filter).
+
+var evenNumberArrayFirstVariant: Array<Int>  = []
+var evenNumberArraySecondVariant: Array<Int> = []
+var evenNumberTuplesArray : [(Int, String)]  = []
+
+//The first variant
+
+for (number, _) in tuplesArray {
+    if number % 2 == 0 {
+        evenNumberArrayFirstVariant.insert(number, at: evenNumberArrayFirstVariant.endIndex)
+    } else {
+        
+    }
```

Comment:
You can just drop `else` here because nothing's happening in this code block

Reply:
i have forgot that we can use if-else block without else option 
Fixed